templates/user/auth/oidc_wellknown.tmpl

```diff
@@ -0,0 +1,10 @@
+{
+    "issuer": "{{AppUrl | JSEscape | Safe}}",
+    "authorization_endpoint": "{{AppUrl | JSEscape | Safe}}login/oauth/authorize",
+    "token_endpoint": "{{AppUrl | JSEscape | Safe}}login/oauth/access_token",
+    "userinfo_endpoint": "{{AppUrl | JSEscape | Safe}}login/oauth/userinfo",
```

Comment:
I would say blocked by #14938 ?

Reply:
Would say remove it here, land this and then suggest the addition in that PR.

Reply:
updated per silverwind suggestion.